setup.py

```diff
@@ -179,6 +179,8 @@ def build_extension(self, ext: CMakeExtension):
     requirements += ["pennylane_lightning=="+version]
 
 suffix = backend.replace("lightning_", "")
+if suffix == "gpu":
+    suffix = suffix[0:].upper()
 suffix = suffix[0].upper() + suffix[1:]
```

Comment:
What's the need here?

Reply:
Just to make sure that the plugin's name is `PennyLane_Lightning_GPU` instead of `PennyLane_Lightning_Gpu`. The later one looks a little bit strange to me.